src/concurrency_manager.cc

```diff
@@ -82,6 +82,16 @@ ConcurrencyManager::InitManagerFinalize()
   }
 }
 
+cb::Error
+ConcurrencyManager::PerformWarmup(
```

Comment:
Each load manager has its own `PerformWarmup()` since each load manager has a different function call for beginning load (i.e. `ChangeConcurrencyLevel()`, `ChangeRequestRate()`, and `InitCustomIntervals()`